### ISSUE TITLE
Show description of issue type 

### ISSUE THREAD

**Issue by samanpwbb** [osmlab/to-fix#63]
A little note (even in the `title` attribute) would make things a little more user friend:

![screen_shot_2015-01-20_at_11_43_44_am](https://cloud.githubusercontent.com/assets/108094/5821396/ded36dec-a099-11e4-8f54-552d9bf8d396.png)

**Comment by nubs** [comment]
Agreed.  It'd be nice to maybe even have a link that goes to further documentation including examples of how to fix them.

**Comment by lxbarth** [comment]
> It'd be nice to maybe even have a link that goes to further documentation including examples of how to fix them.

Right, we'll need per-task briefs saying what to fix and how to fix.

**Comment by aaronlidman** [comment]
I think this should be a larger description in a pane that can be toggled on the right side that someone could keep open to reference while looking at the item on the map.

It could be as long as needed, more than a single sentence and include screenshots, links, whatever to give a full idea of what the problem is, the history, and why it's being worked on.

**Comment by lxbarth** [comment]
@Rub21 @aaronlidman - it's really painful that we don't have this.  We just went through the work of adding http://osmlab.github.io/to-fix/#/task/roundaboutusa and now it's just hanging out there with 0 context. 

@Rub21 - i know these upcoming two weeks are going to be super busy, but can you get this on your backlog? 

Let's just do a giant modal when clicking on a task (also visible when visiting to-fix directly):

![](https://www.evernote.com/shard/s210/sh/ac459117-8f0d-48fb-8835-bb3575d512bd/d9ae7f52e826fceb/res/0faa7dc7-0f74-4452-af0a-478e74a9e22a/skitch.png?resizeSmall&width=832)
